### DISCUSSION TITLE
Cannot see Simplisafe or sensors

### DISCUSSION THREAD

**Discussion by BarneyJing** [homebridge-simplisafe3/homebridge-simplisafe3#387] [Q&A]
Hi - I'm fairly new to Homebridge and Simplisafe

I have managed to set up Simplisafe and can see my sensors in the SS app and have set them for secret alerts

I have installed homebridge-simplisafe3 and have authenticated the account. I have tried child bridge and non child bridge modes

However, I cannot see the Simplisafe base in the Homekit Home app or the sensors (weven when instaling it as an accessory in Home while in child bridge mode)

I am running Homebridge on a mac-mini and the other homebridge plugins are working (Alexa, Ring, Nest, TP-Link etc)

What else can I try?

Many thanks

**Comment by shamoon** [comment]
So if the plug-in is showing your accessories it’s doing it’s job and the problem has to be between homebridge and HomeKit.

I would double-check that the SS bridge is paired (each child bridge needs to be paired separately), and if it is re-do it. When you add the SS bridge in the Home app it should take you through all of the accessories (eg your sensors) so you’ll know it worked.
